pkg/platform/yaml/model.go

```diff
@@ -50,15 +50,19 @@ func EncodeTestcase(tc models.TestCase, logger *zap.Logger) (*NetworkTrafficDoc,
 
 		// maybe we need to concatenate the values
 		return pkg.IsTime(strings.Join(vals, ", "))
-	})...)
+	})
+
+	for _, v := range noiseFieldsFound {
```

Comment:
because noisyfieldfound is an array and it is generated by us by checking whether the response header/body fields contains a Date field .

Reply:
user can write manually also. If you write anything in noise in test-case.yaml it will considered for noise detection. You can try out if you don't get it you can ask again.

Reply:
user can write manually also. If you write anything in noise in test-case.yaml it will considered for noise detection. You can try out if you don't get it you can ask again.

Reply:
So when user manually adds noise fields we need to give option to add regex array as value.

Reply:
the function enclosing this statement is called in capture mode so no testcase file is defined yet

Reply:
in factory.go file, at line 137 i have created assigned noise as empty. we can take input from user like in a header field then i believe we can assign it with something more meaningful !

Reply:
cool got it, I thought it was decodeYaml. But we also have option that user can write noise fields manually in test file and we will consider that . That code may be in decode yaml. What it does is what user has written noise will taken and considered for noise . So we need to add regex support for it. Let me find the code line and send you.

Reply:
this is the line https://github.com/keploy/keploy/blob/0ecb8122f31fbc69ffc0058bf83e58022186339d/pkg/platform/yaml/model.go#L245C3-L245C5
But I guess you have already handled that. Great then I guess this issue is also resolved